tests/bearlib/languages/documentation/DocumentationExtractionTest.py

```diff
@@ -197,19 +204,20 @@ def test_extract_documentation_PYTHON3(self):
         expected = list(DocumentationComment(r.documentation,
                                              r.language,
                                              "doxygen",
+                                             r.indent,
                                              r.marker,
                                              r.range)
                         for r in expected)
 
-        expected.insert(4, DocumentationComment(
+        expected.insert(5, DocumentationComment(
```

Comment:
No I just inserted a new DocumentationComment for testing the indentation, so this had to shift by one position.